packages/tools/devtools/devtools-test-app/src/FluidObject.ts

```diff
@@ -179,39 +179,60 @@ export class AppData extends DataObject {
 		const builder = new SchemaFactory("DefaultVisualizer_SharedTree_Test");
 
 		// TODO: Maybe include example handle
+		class LeafNodeSchema extends builder.object("leaf-node", {
+			value: [builder.boolean, builder.handle, builder.string],
+		}) {}
+
+		class ArrayNodeSchema extends builder.object("array-node", {
+			value: [builder.string, builder.boolean],
+			childLeaf: builder.optional(LeafNodeSchema),
+		}) {}
+
+		class BranchNodeSchema extends builder.object("branch-node", {
+			arrayOrHandleNode: [builder.array(ArrayNodeSchema), builder.handle],
+			numericLeafNode: builder.number,
+		}) {}
 
-		class LeafSchema extends builder.object("leaf-item", {
-			leafField: [builder.boolean, builder.handle, builder.string],
+		class MainBranchA extends builder.object("main-object-a", {
+			numericLeafNode: builder.number,
+			objectNode: BranchNodeSchema,
 		}) {}
 
-		class ChildSchema extends builder.object("child-item", {
-			childField: [builder.string, builder.boolean],
-			childData: builder.optional(LeafSchema),
+		class MainBranchB extends builder.object("main-object-b", {
+			numericLeafNode: builder.number,
+			booleanValue: builder.boolean,
 		}) {}
 
-		class RootNodeSchema extends builder.object("root-item", {
-			childrenOne: builder.array(ChildSchema),
-			childrenTwo: builder.number,
+		class RootFieldA extends builder.object("root-field-a", {
+			value: [builder.boolean, builder.handle, builder.string],
 		}) {}
 
-		const config = new TreeViewConfiguration({ schema: RootNodeSchema });
+		class RootFieldB extends builder.object("root-field-b", {
+			mainObjectNode: builder.optional([MainBranchA, MainBranchB]),
+		}) {}
+
+		const config = new TreeViewConfiguration({
+			schema: [RootFieldA, RootFieldB, builder.string, builder.number],
```

Comment:
Nit: are we overdoing it a bit with the schema? Do we really need _both_ string and number as primitive types here, _and_ two different non-primitive node types, to test/showcase a particular scenario? Or would this be just as effective being just (for example) `[RootFieldA, builder.string]`?

Reply:
The intention was to render multiple schmeas and primitive values as an examle. I can remove the unused schema and make it more concise.